projects/rigaux.org/hexedit/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: https://github.com/pixel/hexedit/archive/refs/tags/{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: pixel/hexedit/tags
+
+dependencies:
+  invisible-island.net/ncurses: 6
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: '*'
+  script: |
+    ./autogen.sh
```

Comment:
do they not provide a tarball with a `./configure`? Generally we prefer that as it is less things that can go wrong.